src/vtvalidate.cpp

```diff
@@ -63,11 +79,16 @@ std::string do_expensive_work(bool louder) {
 // them alive until done.
 // Nan AsyncWorker docs:
 // https://github.com/nodejs/nan/blob/master/doc/asyncworker.md
-struct AsyncHelloWorker : Nan::AsyncWorker {
+struct AsyncValidateWorker : Nan::AsyncWorker {
     using Base = Nan::AsyncWorker;
 
-    AsyncHelloWorker(bool louder, Nan::Callback* cb)
-        : Base(cb), result_{""}, louder_{louder} {}
+    AsyncValidateWorker(v8::Local<v8::Object> buffer, Nan::Callback* cb)
+        : Base(cb), 
+        result_{""},
+        data(node::Buffer::Data(buffer), node::Buffer::Length(buffer)), 
+        tile_() {
```

Comment:
Does `tile(buffer)` work here? I do not think we need to use `Reset()` as we are constructing it here.